gwdetchar/omega/tests/test_html.py

```diff
@@ -280,28 +280,6 @@ def test_fancy_plot():
     assert test.caption is 'test.png'
 
 
-def test_finalize_static_urls(tmpdir):
-    static = os.path.join(str(tmpdir), 'static')
```

Comment:
@duncanmmacleod, I think this line may be the reason for the other build failures. The functions that fail all call `finalize_static_urls`, so they all expect to find a `static` directory, but don't. This is probably my fault for not explicitly including this line in the other unit tests to begin with, apologies for that.

Can you add this in to the other unit tests?

Reply:
Actually, I take that back, `finalize_static_urls` should automatically create this directory. Hmm...